schemas/external/repo/asset.example.1.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Would be useful to include HAL properties in some examples.

Reply:
✅ f39bc9e